### DISCUSSION TITLE
Next Release

### DISCUSSION THREAD

**Discussion by yadvr** [kubernetes-sigs/cluster-api-provider-cloudstack#195] [General]
Kicking this thread to discuss about the next opensource release

**Comment by hrak** [comment]
Any plans to bump to cluster-api 1.3.x for Kubernetes 1.25 support?

btw cluster-api 1.2.8 and 1.3.1 also address some CVE's in cert-manager.

**Comment by rcastrogiovanni** [comment]
Hi @rohityadavcloud , 
any news on this project? Are you still working on this?

Thanks, 
Roberto

**Comment by yadvr** [comment]
Hi @rcastrogiovanni @hrak I've started it here - https://github.com/kubernetes-sigs/cluster-api-provider-cloudstack/issues/311 could you chime in with your thoughts.
